src/components/PagePagination/Pagination.js

```diff
@@ -0,0 +1,71 @@
+import classnames from 'classnames';
+import { usePagination, DOTS } from './PagePagination';
+import './PagePagination.scss';
+
+export const Pagination = totalBookCount => {
+   const {
+    onPageChange,
+    totalCount,
+    siblingCount = 1,
+    currentPage,
+    pageSize,
+    className
```

Comment:
I would recommend taking in a list of books as a prop. It can be called bookList. See Home.js for an example of what a bookList could look like.